go.mod

```diff
@@ -119,3 +119,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20211208200746-9f7c6b3444d2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
+
+replace github.com/GoogleContainerTools/kpt/porch/api => github.com/natasha41575/kpt/porch/api v0.0.0-20221018174516-ddfa4bedc8ec
```

Comment:
I'll take this out and update the dependency when https://github.com/GoogleContainerTools/kpt/pull/3593 merges